src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -617,52 +625,189 @@ - (void)_setupSubscription
 }
 
 #pragma mark Device Interactions
+
+// Helper function to determine whether an attribute has "Changes Omitted" quality, which indicates that past the priming report in
+// a subscription, this attribute is not expected to be reported when its value changes
+//   * TODO: xml+codegen version to replace this hardcoded list.
+static BOOL _MTRAttributeHasChangesOmittedQuality(MTRAttributePath * attributePath)
```

Comment:
If this is static, why do we need the `_MTR` prefixing?

Reply:
I will remove it to match the pattern of the other generated files.

Reply:
Fixed in the last commit.